source/LoginTarget.js

```diff
@@ -36,34 +36,16 @@ export default class LoginTarget {
         return this._form;
     }
 
-    /**
-     * Array of password fields within the associated form
-     * @type {Array.<HTMLInputElement>}
-     * @readonly
-     * @memberof LoginTarget
-     */
-    get passwordFields() {
-        return [...this._passwordFields.values()];
+    get passwordField() {
```

Comment:
Is there a point of having getters and setters here? Why not just `this.passwordField = null` in the constructor?

Reply:
I like having the validation possibilities behind getters and setters. I know I'm not using them to their fullest right now, but I prefer this pattern of working with classes. Having the `this.property` declaration in the constructor feels messy to me (I like having control over knowing when and what each property was set to).

Mostly personal preference, but I have my reasons 😁

Reply:
In this case it just adds unnecessary lines of code, making it both harder to understand and read. Why not add it when/there it's needed instead of everywhere even when it got 0 benefits? Or am I missing something 🤔

Reply:
I use the setters to enforce the set type, and naturally add the getters as well. I disagree that it adds 0 benefits.